### PR TITLE
gemget: new port

### PR DESCRIPTION
**Pull request by sikmir**
#### Description
[**gemget**](https://github.com/makeworld-the-better-one/gemget/) - a command line downloader for the [Gemini protocol](https://gemini.circumlunar.space/).

###### Type(s)
- [ ] bugfix
- [x] enhancement
- [ ] security fix

###### Tested on
macOS 10.13.6
Xcode 10.1

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked that there aren't other open [pull requests](https://github.com/macports/macports-ports/pulls) for the same change?
- [ ] referenced existing tickets on [Trac](https://trac.macports.org/wiki/Tickets) with full URL? 
- [x] checked your Portfile with `port lint`?
- [ ] tried existing tests with `sudo port test`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by sikmir** [comment]
> The CI errors just seem to be issues with checksums, which should be easy to correct.

I just wonder why checksums are different on CI and on my macos.

File: net/gemget/Portfile [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+                        sha256  e45e3181c07b3e2dad8e1317e91a5bbbee4845067e3e3879a585a5254bc6a334 \
+                        size    17273 \
+                    github.com/davecgh/go-spew \
+                        lock    v1.1.1 \
+                        rmd160  7c02883aa81f81aca14e13a27fdca9e7fbc136f7 \
+                        sha256  e85d6afa83e64962e0d63dd4812971eccf2b9b5445eda93f46a4406f0c177d5f \
+                        size    42171
+
+set go_ldflags      "-s -w -X main.version=${version}"
+build.args          -ldflags \"${go_ldflags}\"
```
Comment (harens) [inline net/gemget/Portfile:81]:
```suggestion
build.args          -ldflags '-X main.Version=${version}'
```

Haven't tested this, but it should work.

**Review by harens — COMMENTED** [review]
Thanks @sikmir. I left a minor comment below. Also, would you mind adding a `revision` field?

Following which, I'm happy to merge this. Thank you for opening this PR @sikmir. 👍

**Review by harens — APPROVED** [review]
LGTM. Thanks @sikmir for another great contribution! 🎉It's really appreciated.

**Pull request merged**